lib/allowed-features.js

```diff
@@ -65,7 +65,7 @@ export const isFeatureAllowedForCollectiveType = (collective, feature) => {
 };
 
 export const getFeatureStatus = (collectiveFeatures, feature) => {
-  const featureStatus = collectiveFeatures[feature];
+  const featureStatus = collectiveFeatures?.[feature];
```

Comment:
@sbinlondon added this because it was crashing on all pages with a navbar that don't fetch collective features